client/src/components/Panels/ActivityPanel.vue

```diff
@@ -53,22 +55,26 @@ const emit = defineEmits(["goToAll"]);
             display: flex;
             align-items: center;
             justify-content: space-between;
+            min-height: 2rem;
 
             .activity-panel-heading {
-                margin: 0 !important;
+                margin: 0;
+                padding-left: 0.25rem;
             }
         }
     }
 
     .activity-panel-body {
+        display: flex;
+        flex-direction: column;
         flex-grow: 1;
-        height: 100%;
-        overflow-y: auto;
+        overflow-y: hidden;
```

Comment:
@ElectronicBlueberry The scroll bar is always visible:
![image](https://github.com/galaxyproject/galaxy/assets/8046843/59b3e30d-09d5-4c5f-b1a4-91635d33f227)